docs/security/users-and-teams/auditing.md

```diff
@@ -6,6 +6,10 @@ position: 30
 
 For team members to collaborate in the deployment of software, there needs to be trust and accountability. Octopus Deploy captures audit information whenever significant events happen in the system.
 
+:::warning
```

Comment:
```suggestion
:::hint
```

Reply:
Argh, I meant to accept this suggestion... I'll do it manually 🙂